java/install_jdk

```diff
@@ -3,21 +3,33 @@
 set -e
 set -x
 
-echo "Install glibc-2.21-r2..."
-apk-install libgcc
-wget "https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk" \
-     "https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-bin-2.21-r2.apk" \
-  -P /var/cache/apk
-apk-install --allow-untrusted /var/cache/apk/glibc-2.21-r2.apk /var/cache/apk/glibc-bin-2.21-r2.apk
-/usr/glibc/usr/bin/ldconfig /lib /usr/glibc/usr/lib
+echo "Install glibc-${GLIBC_VERSION}..."
+apk-install libstdc++
+for pkg in glibc-${GLIBC_VERSION} glibc-bin-${GLIBC_VERSION} glibc-i18n-${GLIBC_VERSION}; do
+    curl -sSL https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/${pkg}.apk -o /tmp/${pkg}.apk;
+done
+apk add --allow-untrusted /tmp/*.apk
+rm -v /tmp/*.apk
+( /usr/glibc-compat/bin/localedef --force --inputfile POSIX --charmap UTF-8 C.UTF-8 || true )
```

Comment:
Why the `|| true`? Is this expected to randomly fail? In the context of a repeatable build, either it's needed, it works, and you don't need the `|| true`; or it doesn't work, in which case it seems weird to mask the error and continue.